pwiz_tools/Skyline/Model/Import.cs

```diff
@@ -482,7 +482,8 @@ public bool PreImport(IProgressMonitor progressMonitor, ColumnIndices indices, b
             if (InputType == SrmDocument.DOCUMENT_TYPE.none)
             {
                 InputType =
-                    SmallMoleculeTransitionListCSVReader.IsPlausibleSmallMoleculeTransitionList(lines, Settings) ? SrmDocument.DOCUMENT_TYPE.small_molecules : SrmDocument.DOCUMENT_TYPE.proteomic;
+                    SmallMoleculeTransitionListCSVReader.IsPlausibleSmallMoleculeTransitionList(lines, Settings, 
+                       Program.ModeUI) ? SrmDocument.DOCUMENT_TYPE.small_molecules : SrmDocument.DOCUMENT_TYPE.proteomic;

```

Comment:
This feels wrong - Program.ModeUI in model code? Seems like it should be passed in. What do we do in the command line context?

Reply:
In the command line context it defaults to proteomic mode. Perhaps I'll change it to pass in the modeUI in UI cases and pass in "proteomic" in command line uses. 

Another option for the command-line context could be to use the document type of the open document

Reply:
>> Another option for the command-line context could be to use the document type of the open document
I like it!